### DISCUSSION TITLE
Question: is it possible to select an item inside an array in a loop

### DISCUSSION THREAD

**Discussion by Frolipon** [maizzle/framework#753] [General]
Hello,

I was wondering if it is possible to select a specific item inside an array.
Example:
```html
---
brand: ["windows","apple","linux"]
brand_url: ["https://www.microsoft.com","https://www.apple.com","https://www.linux.org"]
---

<each loop="item, index in page.brand">
  <a href="">
    <img src="images/{{item}}_logo.jpg" />
  </a>
</each>
```

I would like to get `brand_url` at the given `loop.index` to set the `href`. I have tried several things, but I can't make it work.
Is it possible?

I have found a workaround doing this:
```html
<each loop="item_brand, brand_index in page.brand">
  <each loop="item_url, url_index in page.brand_url">
    <if condition="brand_index == url_index">
      <a href="{{item_url}}">
        <img src="images/{{item_brand}}_logo.jpg" />
      </a>
    </if>
  </each>
</each>
```

But I was wondering if there is a simpler way to do it.

**Comment by cossssmin** [comment]
Have you tried [loop meta](https://maizzle.com/docs/tags#loop-meta)? You could use `loop.index` to fetch the `brand_url` with `brand_url[loop.index]`.

Alternatively, if possible in your case, use a single object instead of two arrays, i.e. 

```
brand_urls: [{
  windows: 'https://www.microsoft.com',
  // ...
}]
```

or, clearer:

```
brands: [
  {
    name: 'windows',
    url: 'https://www.microsoft.com',
  },
  // ...
]
```

You could then just loop through that and get `item.name` or `item.url`.

**Comment by Frolipon** [comment]
Hello,

The loop meta does it! I tried it before, but wrote it the wrong way :-/.

I thought about an object, but it isn't really convenient for my case. Here I over simplified the example.

Thanks a lot for the help!
